PTIRelianceLib/PTIRelianceLib.csproj

```diff
@@ -46,6 +46,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="[9.0.1,12.12.0.1)" />
     <PackageReference Include="LibLog" Version="5.0.3" />
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Include="System.Drawing.Common" Version="4.5.0" />
```

Comment:
Does this include work for all targets or does it need a conditional reference?

Reply:
This works for all targets.